script.js

```diff
@@ -0,0 +1,71 @@
+function Books(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function getBooks() {
+  let bookList;
+  if (JSON.parse(localStorage.getItem('bookList') === null)) {
+    bookList = [];
+  } else {
+    bookList = JSON.parse(localStorage.getItem('bookList'));
+  }
+  return bookList;
+}
+
+function addBook(book) {
+  const bookList = getBooks();
+  bookList.push(book);
+  localStorage.setItem('bookList', JSON.stringify(bookList));
+}
+
+function removeBook(title, author) {
+  const bookList = getBooks();
+  const newList = bookList.filter((item) => item.title !== title && item.author !== author);
+  localStorage.setItem('bookList', JSON.stringify(newList));
+}
```

Comment:
- [ ] **Required:** You are doing a fantastic job team. I have observed that your delete button can be improved. It currently works fine when different books are stored, however, if a user store duplicates mistakenly and intend to remove one, all instances are removed.


![same_input_deletes](https://user-images.githubusercontent.com/27709832/176095371-4a4202e0-35e7-4751-97f4-36bf1946e62e.gif)

**_Hint:_**
- You could add an id (integer) that can be set to the length of your booklist and filter the list by the id.